docs/user/rstudio/ide/guide/code/debugging.qmd

```diff
@@ -121,7 +124,7 @@ This overrides RStudio's error handler settings. Trapping every error will quick
 
 ## Using the debugger
 
-Once your code is stopped, the IDE will automatically enter debug mode, in which a variety of tools for inspecting and altering the state of your program become available. We'll look at each in turn.
+Once the code is stopped, the IDE automatically enters debug mode, in which a variety of tools for inspecting and altering the state of your program become available. The debugging tools are displayed in the screenshot below, and further explored in the below sections.
 
 ![](images/using-the-debugger.png){fig-alt="A screenshot of RStudio displaying example R code with a browser(), the R Console executing within the browser() context, the environment pane, and the traceback to the specific function with the browser." width="750"}
 
```

Comment:
In the **Environment pane** below, the included image doesn't show gray values. Consider swapping it out for one that does so that the user has context for "The gray values are promises--in this case, function arguments that haven't been evaluated yet."

Reply:
Fixed!